File: TenIG_MO(Bachelors)_2016.09.01/Programs/FinalSolution/FinalSolution/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace FinalSolution
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            ESM esm = new ESM();
            esm.Show();
        }

        private void button2_Click(object sender, EventArgs e)
        {
            PSM psm = new PSM();
            psm.Show();
        }

        private void button3_Click(object sender, EventArgs e)
        {
            GSM gsm = new GSM();
            gsm.Show();
        }

        private void button4_Click(object sender, EventArgs e)
        {
            NSM nsm = new NSM();
            nsm.Show();
        }
    }
}

File: PPO/2 курс/Project2/WindowsApplication1/WindowsApplication1/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Symptoms.Show()
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Diagnoses.Show()
    End Sub

    Private Sub Button3_Click(sender As Object, e As EventArgs) Handles Button3.Click
        AllSDRelations.Show()
    End Sub

    Private Sub Button4_Click(sender As Object, e As EventArgs) Handles Button4.Click
        Identify.Show()
    End Sub
End Class